sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/BuilderHelper.java

```diff
@@ -130,6 +135,7 @@ private static String getPrimaryEndpointForTokenAuth(String endpoint) {
      */
     private static HttpLogOptions getDefaultHttpLogOptions() {
         HttpLogOptions defaultOptions = new HttpLogOptions();
+        // TODO
```

Comment:
TODO?

Reply:
Yes, this code was commented already but there was no TODO, so I had missed it previously. I don't want to fix it in this PR, it's not critical - all this does is set up the allowlist of header and query string names that are permitted to be logged. Without it, they show up as `REDACTED`. I need to take the time to mine the docs / swagger / etc to determine the list of service headers / query string params that should be permitted.

Reply:
Filed #16408 to track this